docs/miscellaneous.rst

```diff
@@ -11,7 +11,7 @@ Layout of State Variables in Storage
 Statically-sized variables (everything except mapping and dynamically-sized array types) are laid out contiguously in storage starting from position ``0``. Multiple, contiguous items that need less than 32 bytes are packed into a single storage slot if possible, according to the following rules:
 
 - The first item in a storage slot is stored lower-order aligned.
-- Elementary types use only that many bytes that are necessary to store them.
+- Elementary types use only as many bytes that are necessary to store them.
```

Comment:
```suggestion
- Elementary types use only as many bytes as are necessary to store them.
```
I am not a native speaking so I am not sure, but would this be even better?

Reply:
It is, I did somewhat take this literally, and just fix the typo ;)